examples/sha256/table16/message_schedule/subregion2.rs

```diff
@@ -1,6 +1,6 @@
 use super::super::{util::*, AssignedBits, Bits, SpreadVar, SpreadWord, Table16Assignment};
 use super::{schedule_util::*, MessageScheduleConfig, MessageWord};
-use halo2::{arithmetic::FieldExt, circuit::Region, pasta::pallas, plonk::Error};
+use halo2::{circuit::Region, pasta::pallas, plonk::Error};
 use std::convert::TryInto;
 
 /// A word in subregion 2
```

Comment:
Unrelated to the PR, but the clippy warnings here seem suspicious: why are `d` and `g` never read?